web/cookiesCTF/Pages/Task4/Task4notSOLVED.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Task №4</title>
+</head>
+	<body>
+		<h1>Jake is so lonely!</h1>
+		<p>To get the password, send the letter to Jake by POST.</p1>
+		<br></br>
```

Comment:
The `<br>` tag should not be closed: https://www.w3schools.com/tags/tag_br.asp
```suggestion
		<br>
```
(same for other HTML files!)